tests/07reshape5intr

```diff
@@ -31,7 +31,21 @@ do
   echo 1000 > /proc/sys/dev/raid/speed_limit_min
   echo 2000 > /proc/sys/dev/raid/speed_limit_max
   check wait
-  while ! echo check > /sys/block/md0/md/sync_action; do sleep 0.1; done
+
+  max=5
+
+  for ((i = 0 ; i < max ; i++ )); do
+    if [[ echo check > /sys/block/md0/md/sync_action ]]; then
```

Comment:
Please use direct comparison, I understand what you are doing but I would like to keep it as understandable as possible.
```suggestion
if [[ $(echo check > /sys/block/md0/md/sync_action) != 0 ]]; then
```